Assets/_Game/Scripts/UI/UiController.cs

```diff
@@ -0,0 +1,71 @@
+using System.Collections;

```

Comment:
This is most likely gonna be entirely scrapped if you're tasking me with the UI control :P

Reply:
Yes, it's going to get scrapped I planned on that